clang/lib/Driver/ToolChains/Cuda.cpp

```diff
@@ -952,14 +962,6 @@ void CudaToolChain::addClangTargetOptions(
       // Add these flags for .cu SYCL compilation.
```

Comment:
Might be a silly question but can this statement be hit, i.e., can we have `-fsycl` but also `DeviceOffloadingKind != Action::OFK_SYCL`?

Reply:
I think this should now be
```
if (DeviceOffloadingKind == Action::OFK_Cuda && DriverArgs.hasArg(options::OPT_fsycl)) {
  // Add these flags for .cu SYCL compilation.
```
instead of just `if (DriverArgs.hasArg(options::OPT_fsycl))`